src/libpmemfile/sys_util.h

```diff
@@ -46,9 +46,9 @@
  * the program.
  */
 static inline void
-util_mutex_init(pthread_mutex_t *m, const pthread_mutexattr_t *mutexattr)
+util_mutex_init(pthread_mutex_t *m)
```

Comment:
+1 for reviewing pmemfile.
Here is more stuff if you're bored:
https://github.com/pmem/syscall_intercept/pull/2